python/cudf/cudf/core/multiindex.py

```diff
@@ -1609,3 +1610,93 @@ def _level_index_from_level(self, level):
 
     def _level_name_from_level(self, level):
         return self.names[self._level_index_from_level(level)]
+
+    def get_loc(self, key, method=None, tolerance=None):
+        """
+        Get location for a label or a tuple of labels.
+
+        The location is returned as an integer/slice or boolean mask.
+
+        Parameters
+        ----------
+        key : label or tuple of labels (one for each level)
+        method : None
```

Comment:
Do we want to include `tolerance` in the params? Not sure how you lean here since it isn't implemented in either case but is included for `Index.get_loc`

Reply:
I believe this is just to match pandas api: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.MultiIndex.get_loc.html